scripts/process_uf_cells_spleen_0510.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -o errexit
+
+. ./scripts/utils.sh
+
+DATASET='uf_cells_spleen_0510'
+
+main() {
+    get_CLI_args "$@"
+
+    # The variables $BASE, $INPUT, $OUTPUT, $CLOUD_TARGET
+    # come from running the get_CLI_args line above.
+
+    SNAKEMAKE_CMD="snakemake
+        --cores 1
+        --snakefile $BASE/snakemake/$DATASET/Snakefile
+        --directory $BASE/snakemake/$DATASET
+        --config INPUT=$INPUT OUTPUT=$OUTPUT CLOUD_TARGET=$CLOUD_TARGET"
+    
+    eval $SNAKEMAKE_CMD --dryrun
```

Comment:
could you comment why this is needed?

Reply:
Not really needed, just an extra "test" check. If the snakefile fails to produce a proper DAG or has syntax errors they should appear in the dryrun